includes/AMP/Story_Sanitizer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Class Story_Sanitizer.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\AMP;
+
+use AMP_Base_Sanitizer;
+
+/**
+ * Story sanitizer.
+ *
+ * Sanitizer for Web Stories related features.
+ *
+ * @see AMP_Story_Sanitizer
+ *
+ * @since 1.1.0
+ */
+class Story_Sanitizer extends AMP_Base_Sanitizer {
```

Comment:
Is this sanitizer being registered to run with the AMP plugin yet?

Reply:
Due to the prefixing it would need to be copied, actually.
But the logic here is done via the validation callback filters in the Story_Post_Type class, so no need for it.